### PR TITLE
feat: add methods to set and get profiles info

### PR DESCRIPTION
**Pull request by guidota**
## Description

Adds capability to interact with `/profile/{userId}` API to update and retrieve profile information.

### CODE REVIEW

File: src/MessagingAPI.ts [hunk 1]
```diff
@@ -24,6 +24,7 @@ export interface MessagingAPI {
     /** Get total number of unseen messages from all conversations the user has joined */
     getTotalUnseenMessages(): number
 
+    getProfileInfo(userId: string): Promise<{ displayName?: string; avatarUrl?: string }>
```
Comment (moliva) [inline src/MessagingAPI.ts:27]: Might be good to have an alias for this type.

File: src/SessionManagementAPI.ts [hunk 2]
```diff
@@ -6,6 +6,7 @@ export interface SessionManagementAPI {
     getUserId(): SocialId
     getDomain(): string
 
+    setProfileInfo(displayName: String, avatarUrl: String): Promise<void>
```
Comment (moliva) [inline src/SessionManagementAPI.ts:9]: I would set the param to the same object as returned before (and reuse the type here) for maintaining compatibility in the future.

**Review by moliva — COMMENTED** [review]
Leaving some comments to discuss.

File: src/MessagingAPI.ts [hunk 3]
```diff
@@ -24,6 +24,9 @@ export interface MessagingAPI {
...
+
+    getMemberInfo(roomId: string, userId: string): { displayName?: string; avatarUrl?: string }
```
Comment (Julieta11) [inline src/MessagingAPI.ts:29]: And I think this one too:
```suggestion
    getMemberInfo(roomId: string, userId: string): ProfileInfo
```

File: src/MessagingAPI.ts (hunk 1 above)
Comment (Julieta11) [reply src/MessagingAPI.ts:27]: I think this one is missing:
```suggestion
    getProfileInfo(userId: string): Promise<ProfileInfo>
```

**Review by Julieta11 — APPROVED** [review]
Looks great  💅🏻️

**Pull request merged**